pages/faq.tsx

```diff
@@ -1,16 +1,14 @@
 import styled from "@emotion/styled";
 import React, { useState } from "react";
-import arrowDownCircle from "../shared/images/arrow-down-circle.svg";
-import { Section, Separator } from "../shared/layout";
-import { NavigationBar } from "../shared/navigation-bar";
-import mainFaqImage from "./images/main-faq.svg";
+import { Section, Separator } from "../src/components/shared/layout";
+import { NavigationBar } from "../src/components/shared/navigation-bar";
 
 const Container = styled.div`
   background-repeat: no-repeat;
   background-position: top right;
 
   @media screen and (min-width: 768px) {
-    background-image: url(${mainFaqImage});
+    background-image: url(images/faq/main-faq.svg);
```

Comment:
what happens if the image changes? does it generate a unique hash so that cache wont work ?

Reply:
Will only change when `next build` is run again

Reply:
where is it in the doc? I only found https://nextjs.org/docs/basic-features/static-file-serving which requires `next/image`

Reply:
oh I just tried changing the svg myself, had to rebuild before the new one is reflected

Reply:
so it works as in ? dont need to use on a next/image component ?

Reply:
not sure what you mean why "working", but the image can be shown and updates upon builds without using `next/image` yup